navigation/useNavigation.tsx

```diff
@@ -1,97 +1,74 @@
 import {
   NavigationAction,
-  NavigationProp,
-  StackActions,
+  RouteProp,
   useNavigation,
+  useRoute as useRouteNavigation,
 } from "@react-navigation/native";
-import { useEffect, useMemo } from "react";
+import { NativeStackNavigationProp } from "@react-navigation/native-stack";
+import { useEffect } from "react";
 
 import { NavigationParamList } from "../screens/Navigation/Navigation";
 
+// Extend global namespace for type safety
+declare global {
```

Comment:
Should this go in a types.d.ts file instead?

I guess command clicking does bring you to the definition